hazelcast-hibernate5/src/main/java/com/hazelcast/hibernate/local/LocalRegionCache.java

```diff
@@ -296,6 +301,7 @@ public Map asMap() {
     void cleanup() {
         final int maxSize;
         final long timeToLive;
+        final MapConfig config = lookupMapConfig(name, hazelcastInstance);
```

Comment:
I just noticed an issue here. 

A map configuration cannot be changed after a cluster is up and running. However, new configurations can be added ([see doc here](https://docs.hazelcast.org/docs/latest/manual/html-single/#dynamically-adding-data-structure-configuration-on-a-cluster)). That means, in order to change a LocalRegion map config, you have to add another map config and change the local region's name. But the region itself is registered by its name and once its name is changed, this region is going to be unreachable. WDYT?